cmd/really/really.cpp

```diff
@@ -141,16 +141,16 @@ class ReallyServer : public quicr::ServerDelegate
   ~ReallyServer() { server.reset(); };
 
   virtual void onPublishIntent(const quicr::Namespace& quicr_namespace,
+                               const qtransport::TransportContextId context_id,
                                const std::string& /* origin_url */,
-                               bool /* use_reliable_transport */,
```

Comment:
I don't think we should remove this.

Reply:
I'd be inclined to agree, though I have to ask why? The way it is right now the publish intent is sent on the control stream, which is already set to reliable/unreliable.